code/modules/food/recipes_foodsynthesizer.dm

```diff
@@ -0,0 +1,1114 @@
+/datum/category_item/synthesizer/New()
+	..()
+	var/obj/item/weapon/reagent_containers/food/snacks/snacc
+	if(path)
+		snacc = new path()
```

Comment:
You don't actually need to create an instance of the item to use initial() on it a few lines down. You could avoid some performance overhead by just calling initial on the path itself, which byond allows for most vars (including the name and desc).

Reply:
This is how it works for autolathe code, trying to mimic R&D's stuff auto fails because designs aren't category items